java/Oleksii-Riabov/src/com/zoolatech/lecture2/tasks/_1/Calculator.java

```diff
@@ -0,0 +1,134 @@
+package com.zoolatech.lecture2.tasks._1;
+
+import java.util.Scanner;
+
+/**
+ * Define a class that represents a calculator. The class should provide methods
+ * that accept another value and perform addition, subtraction, multiplication and
+ * division operations on a value stored in a calculator instance. Divides by a
+ * zero should be forbidden and ignored. The class should also provide a method to
+ * get a current value. The class should work with both integer and double
+ * numbers (ignore roundoff errors). Assume all operation results fit into the
+ * range of values for a current value type.
+ */
+
+public class Calculator {
+    double value;
+
+    public Calculator(double value) {
+        this.value = value;
+    }
+
+    public double getValue() {
+        return value;
+    }
+
+    public static void numbersEvaluationMethod(double number1, double number2, char c) {
+        switch (c) {
+            case '+' -> addition(number1, number2);
+            case '-' -> subtraction(number1, number2);
+            case '*' -> multiplication(number1, number2);
+            case '/' -> division(number1, number2);
+            default -> throw new IllegalArgumentException("Enter symbol of an operation");
+        }
+    }
+
+    public static void fractionalPartCheck(double number) {
+        if (number % 1 == 0) {
+            System.out.println((int) number);
+        } else {
+            System.out.println(number);
+        }
+    }
+
+    public static void addition (double number1, double number2) {
+        fractionalPartCheck(number1 + number2);
+    }
+
+    public static void subtraction (double number1, double number2) {
+        fractionalPartCheck(number1 - number2);
+    }
+
+    public static void multiplication (double number1, double number2) {
+        fractionalPartCheck(number1 * number2);
+    }
+
+    public static void division (double number1, double number2) {
+        if (number2 == 0) {
+            throw new ArithmeticException("Divide by zero is not allowed!");
+        } else {
+            fractionalPartCheck(number1 / number2);
+        }
+    }
+
+    public static double doubleValue() {
```

Comment:
methods should be defined in the usage order: `doubleValue` and `instructions` should be defined after main